plugins/mcreator-core/features/feature_random_patch_simple.json

```diff
@@ -12,7 +12,6 @@
       "name": "xzSpread",
       "value": 7,
       "min": 0,
-      "max": 16,

```

Comment:
This seems rather unrelated change

Reply:
Abd this seems like something I never changed....

Reply:
Well it is in the PR in the changed files tab, how it came there, that I do not know